blaze-client/src/main/scala/org/http4s/client/blaze/bits.scala

```diff
@@ -21,12 +21,13 @@ private[blaze] object bits {
 
   val ClientTickWheel = new TickWheelExecutor()
 
-  def getExecutor(config: BlazeClientConfig): (ExecutorService, Task[Unit]) = config.customExecutor match {
-    case Some(exec) => (exec, Task.now(()))
-    case None =>
-      val exec = threads.newDaemonPool("http4s-blaze-client")
-      (exec, Task.delay(exec.shutdown()))
-  }
+  def getExecutionContext(config: BlazeClientConfig): (ExecutionContext, Task[Unit]) =
+    config.customExecutionContext match {
+      case Some(ec) => (ec, Task.now(()))
+      case None =>
+        val exec = threads.newDaemonPool("http4s-blaze-client")
```

Comment:
It seems like this should use `org.http4s.util.threads.DefaultExecutionContext` as it is in core so should be available.